frontend/components/explore/droplets-grid.tsx

```
@@ -30,18 +30,25 @@ export async function DropletsGrid({
   const user = await getCurrentUser();
   let enrolledDropletIds: number[] = [];
   let completedLessonIds: number[] = [];
+  let archivedDropletIds: number[] = [];
+  let favoritedDropletIds: number[] = [];
```

ESLint

[error] 'favoritedDropletIds' is never reassigned. Use 'const' instead.

(prefer-const)